sdk/analytics/build.gradle

```diff
@@ -19,6 +19,7 @@ android {
 dependencies {
     implementation project(":sdk:core")
     testImplementation project(":sdk:fixtures")
+    testImplementation KotlinX.coroutines.test
```

Comment:
@dan-peluso I had to run `./gradlew refreshVersionsMigrate --mode=VersionsPropertiesOnly` to figure out the right syntax, figured I'd just push a commit once i'd figured it out though.